server/server.go

```diff
@@ -106,12 +106,12 @@ type TLSConfig struct {
 // New creates a new Server which uses the supplied backends
 // for managing data.
 func New(dependencies *Dependencies, config *Config) (*Server, error) {
-	tokenEncoder := dependencies.TokenEncoder
-	if tokenEncoder == nil {
-		tokenEncoder = encoder.NewBase64Encoder()
+	tokenEncrypter := dependencies.TokenEncrypter
+	if tokenEncrypter == nil {
```

Comment:
By the way, why do we check these and not, say, datastore? Are these checks useful, or should we just fail fast?